recipes/raylib/all/conanfile.py

```diff
@@ -127,6 +134,9 @@ def package_info(self):
         elif self.settings.os == "Windows":
             self.cpp_info.system_libs.append("winmm")
 
+        # Some useful files are not packaged by default
+        self.cpp_info.resdirs = ["res/include"]
```

Comment:
```suggestion
        self.cpp_info.resdirs = ["res/include"]
        if self.conf.get("user.raylib:include_res", , default=False, check_type=bool):
            self.cpp_info.includedirs.append("res/include")
```

So users can use res folder directly without affecting the package ID.